portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase

```diff
@@ -209,6 +209,22 @@ definition {
 		}
 	}
 
+	@description = "Ensure that the URL is correct when there is an Asset Publisher in a page."
```

Comment:
@description = "This is a test for LPS-128196. View the URL when there is an Asset Publisher in a page."